src/platform/packages/shared/kbn-unified-field-list/src/components/field_list_grouped/field_list_grouped.test.tsx

```diff
@@ -76,6 +78,27 @@ describe('UnifiedFieldList FieldListGrouped + useGroupedFields()', () => {
     hookParams: Omit<GroupedFieldsParams<DataViewField>, 'services'>;
   }
 
+  function mountWithRTL({ listProps, hookParams }: WrapperProps) {
+    const Wrapper: React.FC<WrapperProps> = (props) => {
+      const {
+        fieldListFiltersProps,
+        fieldListGroupedProps: { fieldGroups },
+      } = useGroupedFields({
+        ...props.hookParams,
+        services: mockedServices,
+      });
+
+      return (
+        <>
+          <FieldListFilters {...fieldListFiltersProps} />
+          <FieldListGrouped {...props.listProps} fieldGroups={fieldGroups} />
+        </>
+      );
+    };
+
+    render(<Wrapper hookParams={hookParams} listProps={listProps} />);
```

Comment:
I'd go for shortcuts everywhere when your naming matches, but from previous PRs I have a hunch you just like to have them explicitly written - if that's the case, let me know and I'll ignore it further 😁 Don't want to coerce your style if it's not really relevant.

```suggestion
          <FieldListGrouped {...props.listProps} fieldGroups />
        </>
      );
    };

    render(<Wrapper hookParams listProps />);
```

Reply:
i don't think this is the same - when the value is not specified it's just true, it's not the same behaviour as in objects 👀